test/convert-source-map.js

```diff
@@ -4,30 +4,45 @@
 var test = require('tap').test
   , generator = require('inline-source-map')
   , convert = require('..')
+  , decodeBase64 = typeof Buffer.from ?
+    function decodeBase64(base64) {
```

Comment:
This is super hard to parse. Please declare these methods separately and refer to them by name .. most likely this becomes a one-liner then

Reply:
Done!